tmux/tmux.go

```diff
@@ -12,6 +12,53 @@ import (
 	"github.com/joshmedeski/sesh/dir"
 )
 
+// Interface for using the tmux cli to interact with a tmux server.
+type Interface interface {
+	GetSession(string) (TmuxSession, error)
+}
+
+// Command is an implementation of the Interface interface providing a means of
+// interacting with the tmux cli.
+type Command struct{}
```

Comment:
Can you add a test to show how this would benefit the codebase? I don't see it currently providing any value.

Reply:
This is setup code that would be used in future refactoring. I'll drop it for now to avoid any confusion.